extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1251,6 +1255,8 @@ static public function getLocalEnvURL( $url, $forcedEnv = null ) {
 		) ? $forcedEnv : $wgWikiaEnvironment;
 
 		// put the address back into shape and return
+		// we need to change 'https' to 'http' for stable/preview/verify/sandbox environments cause
+		// we do not have valid ssl certificate for these subdomains
 		switch ( $environment ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)